tests/Smoke/StdinTest.php

```diff
@@ -45,7 +45,7 @@ public function testFixingStdin()
                     $fileResult->getError()
                 ),
                 'output' => str_ireplace(
-                    str_replace('/', DIRECTORY_SEPARATOR, 'PHP-CS-Fixer/tests/Fixtures/Integration/set/@PSR2.test-in.php'),
+                    str_replace('/', DIRECTORY_SEPARATOR, basename(realpath($cwd)) . '/tests/Fixtures/Integration/set/@PSR2.test-in.php'),
```

Comment:
Can we use `$inputFile` as it's exactly the path suffix?

Reply:
yes we do ;)